src/datepicker/calendar.js

```diff
@@ -88,6 +88,11 @@ export default class Calendar extends React.Component<
       date: dateInView,
       quickSelectId: null,
       rootElement: null,
+      time: Array.isArray(this.props.value)
```

Comment:
it may be useful for readability to pull the multiple ternaries into a mutable variable

Reply:
```js
let time = [];
if (Array.isArray(this.props.value) {
  time = this.props.value;
} else if (this.props.value) {
  time = [this.props.value];
}
```